File: snippets/ruby/batch_webhooks.rb
require "MailchimpMarketing"

mailchimp = MailchimpMarketing::Client.new
mailchimp.set_config({
  :api_key => ENV["API_KEY"],
  :server => ENV["API_SERVER"]
})

url = "https://example.com/your-webhook-url"

response = mailchimp.batchWebhooks.create url
puts response

File: snippets/php/batch_webhooks.php
require_once('/path/to/MailchimpMarketing/vendor/autoload.php');

$mailchimp = new MailchimpMarketing\ApiClient();

$mailchimp->setConfig([
    'apiKey' => 'YOUR_API_KEY',
    'server' => 'YOUR_SERVER_PREFIX'
]);

try {
    $response = $mailchimp->batchWebhooks->create([
        "url" => "https://example.com/your-webhook-url",
    ]);

    echo $response;
} catch (\MailchimpMarketing\ApiException $e) {
    echo $e->getMessage();
}